version/version.go

```diff
@@ -18,4 +18,4 @@ package version
 // Changes may cause incorrect behavior and will be lost if the code is regenerated.
 
 // Number contains the semantic version of this SDK.
-const Number = "v52.3.0"
```

Comment:
What happened to `v52.4.0`?

Reply:
for Release v52.3.0, v52.4.0, v52.5.0. I use the Dapeng's go sdk release tool, it seems in  v52.3.0, this version is updated automatically, but in v52.4.0 it's not updated. Both of us missed check it. 
In today's release, I found this issue and added the latest commit. I will talk with Dapeng then he is back and improve the tool.